CHANGELOG.md

```diff
@@ -10,12 +10,17 @@ modified Semantic Versioning scheme. See the "Versioning scheme" section of the
 
 ## [Unreleased]
 ### Added
+- RIGA-10: Added paragraph translation sync settings to person and location features.
```

Comment:
Is this included in this PR? I didn't spot it

Reply:
Nope, messed up the rebase. Thanks!